cordapp-cpk/src/test/resources/transitive-remote-cordapps/src/main/kotlin/com/example/transitives/ExampleContract.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.transitives
+
+import net.corda.v5.ledger.contracts.Contract
+import net.corda.v5.ledger.transactions.LedgerTransaction
+import org.apache.commons.io.IOUtils
+import java.io.ByteArrayInputStream
+
+class ExampleContract : Contract {
+    override fun verify(ltx: LedgerTransaction) {
+        ByteArrayInputStream("Hello Corda!".toByteArray()).use { input ->
+            IOUtils.copy(input, System.out)
+        }
```

Comment:
I assume the point of this is to simply use `commons-io:commons-io:$commons_io_version` somehow so that it appears in OSGi metadata

Reply:
It's a "something" to appear in both the OSGi metadata _and_ the CPK's `lib/` folder, yes.